code/modules/projectiles/ammo_datums.dm

```diff
@@ -1174,13 +1174,16 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	icon_state = "stun"
 	hud_state = "taser"
 	hud_state_empty = "battery_empty"
-	damage = 120
-	penetration = 0
+	damage = 10
+	penetration = 100
 	damage_type = STAMINA
 	flags_ammo_behavior = AMMO_ENERGY
 	max_range = 15
 	accurate_range = 10
 	bullet_color = COLOR_VIVID_YELLOW
+	
+/datum/ammo/energy/taser/on_hit_mob(mob/M,obj/projectile/P)
+	staggerstun(M, P, weaken = 1, stagger = 13, slowdown = 1, knockback = 1)
```

Comment:
Make it just flat out stun

Reply:
> Make it just flat out stun

Youre assuming Im competent, let me look at how the flash does things. This is literally a bandaid for something that ever has to be used to stop people from murdering people with it then going its  taser it cant kill people" multiple times to admins so they get away with it.